opentreemap/treemap/instance.py

```diff
@@ -158,9 +158,9 @@ class Instance(models.Model):
         'BenefitCurrencyConversion', null=True, blank=True)
 
     """ Center of the map when loading the instance """
-    bounds_obj = models.OneToOneField(InstanceBounds,
-                                      on_delete=models.CASCADE,
-                                      null=True, blank=True)
+    bounds = models.OneToOneField(InstanceBounds,
```

Comment:
I think this still needs another migration that removes the old bounds field first.